index.html

```diff
@@ -41,8 +41,16 @@ <h3 class="error-message hidden"><img class="warning-label" src="./assets/warnin
     </section>
     <aside class="container right-container">
       <h1 class="past-activites">Past Activities</h1>
-      <p class="havent-logged">You haven't logged any activities yet.</p>
-      <p class="complete-form">Complete the form to the left to get started!</p>
+      <section class="past-activities-card" id="past-activities-card">
```

Comment:
Excellent job placing the elements in their own container. Really helped with styling!